mla/src/crypto/aesgcm.rs

```diff
@@ -1,24 +1,27 @@
 use crate::Error;
 
-use aes::{
-    block_cipher::{BlockCipher, NewBlockCipher},
-    Aes256,
-};
-use aes_ctr::stream_cipher::{NewStreamCipher, StreamCipher, SyncStreamCipherSeek};
+use aes::{Aes256, BlockEncrypt, NewBlockCipher};
+
 use generic_array::{typenum::U16, GenericArray};
 use ghash::{
     universal_hash::{NewUniversalHash, UniversalHash},
     GHash,
 };
-
 pub use subtle::ConstantTimeEq;
 
+use ctr::{
+    cipher::{NewCipher, StreamCipher, StreamCipherSeek},
+    Ctr,
+};
+
+type Aes256Ctr = ctr::Ctr128BE<aes::Aes256>;
+
 pub const BLOCK_SIZE: usize = 128 / 8;
 pub const TAG_LENGTH: usize = BLOCK_SIZE;
 
 // Inspired from RustCrypto's AesGcm implementation
 pub struct AesGcm256 {
-    cipher: aes_ctr::Aes256Ctr,
+    cipher: Ctr<Aes256, ctr::flavors::Ctr128BE>,
```

Comment:
You might want to use the type defined above here (`Aes256Ctr`)

Reply:
Indeed, fixed